misc/twindle-web/dark-theme.css

```diff
@@ -0,0 +1,118 @@
+:root {
+  --primary-heading-text: #eee;
+  --primary-background-color: #060606;
+  --secondary-background-color: #0c0c0b;
+  --scrollbar-track-color: #121211;
+  --scrollbar-thumb-color: #3b3c38;
+}
+
+html,
+body {
+  background: var(--primary-background-color);
+  scrollbar-color: var(--scrollbar-thumb-color) var(--scrollbar-track-color);
+}
+
+body::-webkit-scrollbar-track {
+  background-color: var(--scrollbar-track-color);
+}
+
+body::-webkit-scrollbar-thumb {
+  background-color: var(--scrollbar-thumb-color);
+}
+
+.line {
+  background-color: rgba(255, 255, 255, 0.2);
+}
+
+header {
+  background-color: var(--primary-background-color);
+}
+
+.sticky {
+  box-shadow: 0 2px 4px #353632;
+}
+
+.nav-links,
+.logo {
+  color: #fff;
+}
+
+.main-nav li a {
+  color: #fff;
+}
+
+.logo {
+  color: #dbdbdb;
+}
+

```

Comment:
Here too. Please see where there is color define, we can use CSS variable instead to minimize rewriting the code